src/Pages/AdminTaskTable/components/ViewPayloadModal/ViewPayloadModal.tsx

```diff
@@ -9,44 +9,49 @@ import {
   Tabs,
 } from '@patternfly/react-core';
 import { createUseStyles } from 'react-jss';
-import { createRef, useMemo, useState } from 'react';
+import { createRef, useEffect, useMemo, useState } from 'react';
 import { AdminTask } from '../../../../services/AdminTasks/AdminTaskApi';
 import AdminTaskInfo from './components/AdminTaskInfo';
 import ReactJson from 'react-json-view';
 import Hide from '../../../../components/Hide/Hide';
 import { useFetchAdminTaskQuery } from '../../../../services/AdminTasks/AdminTaskQueries';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ADMIN_TASKS_ROUTE } from '../../../../Routes/useTabbedRoutes';
 
 const useStyles = createUseStyles({
   jsonView: {
     fontSize: '12px',
   },
 });
 
-export interface ViewPayloadProps {
-  setClosed: () => void;
-  open: boolean;
-  uuid?: string;
-  status?: AdminTask['status'];
-}
-
 export interface TabData {
   title: string;
   data: object;
   contentRef: React.RefObject<HTMLElement>;
 }
 
-const ViewPayloadModal = ({ uuid, status, open, setClosed }: ViewPayloadProps) => {
+const ViewPayloadModal = () => {
+  const { taskUUID: uuid } = useParams();
   const classes = useStyles();
+  const navigate = useNavigate();
+
+  const onClose = () => navigate(`/settings/content/${ADMIN_TASKS_ROUTE}`);
```

Comment:
In `fec.js` the `appUrl` is `'/insights/content'` When closing a modal from that url, it redirects to a different page (`/settings/content`). The contents of the main part of the page are the same, but the menu on the left is different.

Reply:
ohhh. good catch. I should also see if this can be an imported value.